ds248x/ds248x.go

```diff
@@ -154,6 +161,70 @@ func (d *Dev) Search(alarmOnly bool) ([]onewire.Address, error) {
 	return onewire.Search(d, alarmOnly)
 }
 
+// ChannelSelect function is for selecting one of eight 1-w channels on DS2482-800. 
+// On other chips it does nothing. Function silently limits channel selection between
+// 0 and 7. It is expected that application keeps track of
+// with 1-w device is connected to with channel.
+//
+// Communication error is returned if present.
+//
+func (d *Dev) ChannelSelect(ch int) (err error) {
+	switch d.isDS248x {
+	case isDS2482x100:
+
+	case isDS2482x800:
+		if ch < 0 {
+			ch = 0
+		}
+		if ch > 7 {
+			ch = 7
+		}
+		csc := []byte{cscIO0w, cscIO1w, cscIO2w, cscIO3w, cscIO4w, cscIO5w, cscIO6w, cscIO7w}
```

Comment:
Make it a global, otherwise it'll be constructed at each call.

Reply:
Sorry it's a habit from memory restrained machines.